src/backend/sev/mod.rs

```diff
@@ -14,8 +14,10 @@ use data::{
     dev_kvm, dev_sev, dev_sev_readable, dev_sev_writable, has_reasonable_memlock_rlimit,
     kvm_version, sev_enabled_in_kernel, CPUIDS,
 };
+
+use kvm_bindings::bindings::kvm_enc_region;
+
 use kvm_ioctls::VmFd;
```

Comment:
```suggestion

use kvm_bindings::bindings::kvm_enc_region;
use kvm_ioctls::VmFd;
```

Reply:
OK, will do, thanks to everyone for patience :-)